docs/scripts/build.js

```diff
@@ -0,0 +1,78 @@
+const nunjucks = require('../../index');
+const listMarkdownFiles = require('../lib/list-markdown');
+const templateMeta = require('../lib/template-metadata');
+const renderMarkdown = require('../lib/markdown-to-html');
+const getHeadings = require('../lib/get-headings');
+const saveFile = require('../lib/save-file');
+const copyFile = require('../lib/copy-file');
+
+const baseDir = 'docs/';
+const distDir = 'docs/files/';
+
+// setup nunjunks environment
+const renderer = new nunjucks.Environment(
+	new nunjucks.FileSystemLoader(baseDir, {
+		noCache: true,
+		watch: false
+	}),
+	{autoescape: false}
+);
+
+const config = {
+	site: {
+		baseurl: ''
```

Comment:
Yes, this is also necessary for the translations, otherwise it won't have the correct paths.